### DISCUSSION TITLE
Software Dimming for "Hardware (Apple)" control method

### DISCUSSION THREAD

**Discussion by HalfWhitt** [MonitorControl/MonitorControl#1490] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

I have two 27" LED Cinema Displays (the DisplayPort model) connected to my M1 Studio via USB-C to DisplayPort connectors. MonitorDisplay correctly sees both monitors and adjusts brightness using Apple's hardware protocol. But since it's not DDC, additional software dimming isn't available.

### Describe the solution you'd like

I'd like to be able to use software dimming in addition to Apple hardware brightness control, to be able to dim a screen to black.

### Describe alternatives you've considered

While they don't even have a contrast adjustment, I believe (due to experience with [Lunar](https://lunar.fyi/)) that these monitors' brightness can still be controlled via DDC. If that's correct, then an option to switch control method to DDC would (I imagine) also make software dimming available.

### Anything else?

![image](https://github.com/MonitorControl/MonitorControl/assets/50283671/b16d16a6-9da3-4412-a9f6-86231ddc2532)

**Comment by waydabber** [comment]
Hi @HalfWhitt - this feature was implemented in [BetterDisplay](https://betterdisplay.pro). MonitorControl's feature set is final and no more updates are planned. BetterDisplay allows for extra software dimming with Apple displays as well and you can also force the display to be treated as third party with DDC control. All these features are free in the app.
